tests/unit/LHNOrderTest.js

```diff
@@ -248,22 +248,24 @@ describe('Sidebar', () => {
                     [`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}1`]: fakeReport1Actions,
                 }))
 
-                // THEN the component should be rendered with an item for the fake report
+                // Then the component should be rendered with an item for the fake report
                 .then(() => {
                     expect(sidebarLinks.toJSON()).not.toBe(null);
                     expect(sidebarLinks.toJSON().children.length).toBe(2);
                     expect(sidebarLinks.getAllByText('ReportID, One')).toHaveLength(1);
-                });
+                })
+
+                .then(sidebarLinks.unmount);
```

Comment:
Ah, I didn't realize that cleanup() could be called. I can try that. The problem was that when `Onyx.clear()` was called, it would cause a lot of errors in the console because of `null` data in Onyx.

Reply:
Does this need to be addressed? It sounds like we were good to remove these